server/db/Seeder.js

```diff
@@ -15,14 +15,18 @@ class Seeder {
     let adoptablePets = contents.data.adoptablePets;
 
     try{
-      petTypes.forEach(async (petType) => {
-          let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3)';
+      for( let x = 0; x < petTypes.length; x++){
+          let petType = petTypes[x];
+          let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3) RETURNING id';
           let result = await pool.query(query, [petType.type, petType.imgURL, petType.description]);
-        });
-
-      adoptablePets.forEach( async (pet) => {
-        let idArr = await pool.query('INSERT INTO adoptable_pets (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING id', [pet.name, pet.img_url, pet.age, pet.vaccination_status, pet.adoption_story, pet.available_for_adoption, pet.pet_type_id])
-      })
+          console.log(result.rows[0]);
+        }
+
+      for( let x = 0; x < adoptablePets.length; x++){
+        let pet = adoptablePets[x];
+        let idArr = await pool.query('INSERT INTO adoptable_pets (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING id', [pet.name, pet.img_url, pet.age, pet.vaccination_status, pet.adoption_story, pet.available_for_adoption, pet.pet_type_id]);
```

Comment:
Instead of supplying the pet_type_id within the data file, I would recommend taking a look at the seeder from this clinic as an example
https://github.com/LaunchAcademy/one-to-many-associations-express-and-postgres-facilitation/blob/main/server/db/Seeder.js

where we can insert the pet types, then query the database for those records directly so we aren't assuming any foreign keys